README.md

```diff
@@ -212,6 +212,13 @@ You can even run python scripts like this:
   shell: pixi run python {0}
 ```
 
+Using Windows? Try `powershell` or `bash`:
```

Comment:
```suggestion
If you want to use PowerShell, you need to specify `-Command` as well.
```

We already mentioned how it works with `bash` above. Also `pwsh` is technically platform agnostic 😅